### DISCUSSION TITLE
Consider adding `smart-tabs` to `no-mixed-spaces-and-tabs` rule

### DISCUSSION THREAD

**Discussion by isBatak** [infinum/js-linters#38] [General]
Current rule: https://github.com/eslint/eslint/blob/main/conf/eslint-recommended.js#L47
Proposal: `"no-mixed-spaces-and-tabs": ["error", "smart-tabs"]`

Problem explained here:
https://github.com/prettier/prettier/issues/4199

eslint docs:
https://github.com/eslint/eslint/blob/dd58cd4afa6ced9016c091fc99a702c97a3e44f0/docs/rules/no-mixed-spaces-and-tabs.md#options

![image](https://user-images.githubusercontent.com/4128883/140516529-c1500a84-e39a-4162-9d11-30dd7c95752d.png)

**Comment by andreicek** [comment]
Noice
